addons/mail/static/src/js/many2one_avatar_user.js

```diff
@@ -26,6 +26,9 @@ odoo.define('mail.Many2OneAvatarUser', function (require) {
     const { Many2OneAvatar } = require('web.relational_fields');
     const session = require('web.session');
 
+    var core = require('web.core');
+    var _lt = core._lt;
```

Comment:
just import `_lt` from `web.core` in line 24:
```js
const { _t, _lt } = require('web.core'); 
```

Reply:
oky...